src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  box-sizing: border-box;
```

Comment:
Avoid using the `*` selector as it impacts performance. It's better to set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for the browser.